universe/marathon.json.mustache

```diff
@@ -60,23 +60,23 @@
             "protocol": "tcp",
             "name": "dispatcher",
             "labels": {
-                "VIP_0": "spark-dispatcher:7077"
+                "VIP_0": "{{service.name}}:7077"
```

Comment:
This is a breaking change for existing customers. This should be `{{service.name}}-dispatcher`. cc @vishnu2kmohan

Reply:
Good catch. We should maintain this for our 2.2.1-x packages but do we want to keep this when we bump base-tech to spark 2.3?

Reply:
Actually, this would only maintain behavior for clients who installed spark packages as `spark`. If they named it anything else (eg foldered names), this will still break once they update the package version.